SignalServiceKit/tests/Account/SignedPreKeyDeletionTests.m

```diff
@@ -28,10 +28,9 @@ - (void)tearDown {
 }
 
 - (void)testSignedPreKeyDeletion {
-    [[OWSPrimaryStorage sharedManager].dbReadWriteConnection
-        readWriteWithBlock:^(YapDatabaseReadWriteTransaction *transaction) {
-            [transaction removeAllObjectsInCollection:OWSPrimaryStorageSignedPreKeyStoreCollection];
-        }];
+    [[OWSPrimaryStorage sharedManager].dbReadWriteConnection readWithBlock:^(YapDatabaseReadTransaction *transaction) {
```

Comment:
Now that we're so reliably cleaning out the DB between tests, I'm ambivalent about the signal v noise of these sanity asserts. TIOLI though

Reply:
Right - I haven't bothered removing them yet. We could.